parts/KeyController.cpp

```diff
@@ -78,6 +87,12 @@ void KeyController::PutNiceKeyName(unsigned char key)
 		case 0xd:
 			std::cout << "Enter";
 			break;
+		case GLUT_KEY_UP | SPECIAL_KEY_MASK:
+			std::cout << "Up Arrow";
+			break;
+		case GLUT_KEY_DOWN | SPECIAL_KEY_MASK:
+			std::cout << "DOwn Arrow";
```

Comment:
Typo?

Reply:
D'oh... one of my shift keys is not well molded and sometimes gets stuck for an extra keystroke. One of these days I'll get bothered enough to fix that XD....